clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,15 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateTOptions{mBlockSizeBytes=" + mBlockSizeBytes + ",mRecursive=" + mRecursive
```

Comment:
Or you can strip the spaces for test. Also, if we do not upgrade guava, what about we add a similar util method, which will be handy for all toString methods. And we can easily replace it when we feel it's time to upgrade Guava.